dcos/subcommand.py

```diff
@@ -39,13 +39,39 @@ def command_executables(subcommand, dcos_path):
     return executables[0]
 
 
+def list_package_exectuables(package, dcos_path):
```

Comment:
this function doesn't get all executables.  it only gets those that are dcos commands.  So maybe call this "get_package_commands"